frontend/src/app/components/about/about.module.ts

```diff
@@ -25,10 +26,14 @@ export class AboutRoutingModule { }
   imports: [
     CommonModule,
     AboutRoutingModule,
-    SharedModule,
+    SharedModule
```

Comment:
Leave the comma, so that it won't cause a extra row git change when another module import is added